### ISSUE TITLE
PDF download missing

### ISSUE THREAD

**Issue by codemicro** [GenderDysphoria/GenderDysphoria.fyi#102]
When requesting https://genderdysphoria.fyi/en.pdf, the following error is shown.

```
HTTP/2 404 Not Found
content-type: application/xml
date: Thu, 23 Jun 2022 09:00:42 GMT
server: AmazonS3
x-cache: Error from cloudfront
via: 1.1 8424840dfb521b34b0bba436441f1c36.cloudfront.net (CloudFront)
x-amz-cf-pop: LHR61-P2
x-amz-cf-id: b9qFZvJFhHYCRhWd2qVTOoieFoQOMd6JsVlfCzHmIcMq-hfI7Pz1Nw==
X-Firefox-Spdy: h2

<?xml version="1.0" encoding="UTF-8"?>
<Error>
    <Code>NoSuchKey</Code>
    <Message>The specified key does not exist.</Message>
    <Key>en.pdf</Key>
    <RequestId>6W6N7WPZ41F6SNYD</RequestId>
    <HostId>i0DSBDollZk2NAtTDXnnzAsRTnGDabLAcqQG4dHU++bnoaJ85wqUd6eaezmUvVVhaeBlYPu5/+k=</HostId>
</Error>
```

(Love the website, by the way - it's really tricky to put into words how much of a help it's been to me, so, thanks so much :) )

**Comment by ErdaradunGaztea** [comment]
Actually the original title is correct, the download doesn't work for any language, as far as I checked.

**Comment by codemicro** [comment]
Honestly I probably should have checked that beforehand 🤦‍♀️

**Comment by Twipped** [comment]
Sorry, this is what happens when I do a publish mid-project. I’ll try to get it fixed today.

**Comment by Twipped** [comment]
Fixed with c015fb8. Still trying to get the non-english pdfs building.
